src/styles/02-molecules/search-bar/_search-bar.scss

```diff
@@ -39,6 +39,10 @@ $search-bar--breakpoint: 856px;
 
     flex: 1 1 70%;
     min-height: 40px; // Accommodates the border for the search-bar without parameters
+    
+    &--error::placeholder { //TODO: IE11 support
```

Comment:
@helenvholmes can I get 👀 on this?  (https://caniuse.com/#search=%3A%3Aplaceholder) 
(this is trying to deal with the special ResearchNow error situation - http://themetronome.co/components/oh-31-header-with-search/?fresh=true)

PS if this seems like a bigger thing, let's file a bug.

Reply:
Looks like styling it with red isn't possible in IE11, is that what I'm understanding?

I say we follow up with Willa and ask if just having the text without a styled color is a graceful enough degradation for IE11. Unfortunately I'm not a CSS expert when it comes to IE :(

Reply:
Styling a placeholder isn't possible with IE11, and I think needs the extra `webkit-` `moz-` whatever tags for the older stuff.  (I'll add those in, it's not a big lift and we'll need them eventually, I bet)

Reply:
We don't support older than IE11. Since IE11 styling isn't supported, I think it's moot—we probably ship what's here and let the browser style it natively.

Reply:
Oh sorry I mean ... older than current, newer than IE11 stuff, like Edge 18 (needs `-ms-`)

Reply:
Sounds like something we should make sure our webpack config handles—is this blocking for MVP?

Reply:
Nope, let's throw it on the pile 
https://github.com/NYPL/nypl-design-system/projects/2#card-31347325